libres/lib/analysis/ies/ies_enkf.cpp

```diff
@@ -330,37 +233,128 @@ void ies_enkf_updateA(
     for (int i = 0; i < ens_size; i++) {
         matrix_iadd(X, i, i, 1.0);
     }
-    if (dbg)
-        matrix_pretty_fprint_submat(X, "X", "%11.5f", log_fp, 0, m_ens_size - 1,
-                                    0, m_ens_size);
-
-    /* COMPUTE NEW ENSEMBLE SOLUTION FOR CURRENT ITERATION  Ei=A0*X (Line 11)*/
-    matrix_pretty_fprint_submat(A, "A^f", "%11.5f", log_fp, 0, m_state_size, 0,
-                                m_ens_size);
-    ies_enkf_linalg_extract_active_A0(data, A0, log_fp);
-    matrix_matmul(A, A0, X);
-    matrix_pretty_fprint_submat(A, "A^a", "%11.5f", log_fp, 0, m_state_size, 0,
-                                m_ens_size);
 
     /* COMPUTE ||W0 - W|| AND EVALUATE COST FUNCTION FOR PREVIOUS ITERATE (Line 12)*/
     matrix_type *DW = matrix_alloc(ens_size, ens_size);
     matrix_sub(DW, W0, W);
-    teclog(W, D, DW, "iesteclog.dat", ens_size, iteration_nr, rcond, nrsing,
-           nrobs_inp);
-    teccost(W, D, "costf.dat", ens_size, iteration_nr);
 
-    ies_enkf_data_fclose_log(data);
+    std::vector<double> costJ(ens_size);
+    double costf = 0.0;
+    for (int i = 0; i < ens_size; i++) {
+        costJ[i] = matrix_column_column_dot_product(W, i, W, i) +
+                   matrix_column_column_dot_product(D, i, D, i);
+        costf += costJ[i];
+    }
+    costf = costf / ens_size;
+
+    teclog(W, D, DW, "iesteclog.dat", ens_size, iteration_nr, nrobs_inp);
```

Comment:
Not really relevant, but what is a `tec`?

Reply:
This is call to produce output for the [Tecplot](https://www.tecplot.com/) application. I wanted to remove it - but I am to polite a guy ....

Reply:
First time I hear of Tecplot, thanks!
If it's not used we should definitely remove it, no need to be polite about it :)

Reply:
I'll ask around a bit.

Reply:
> If it's not used 

@geirev used to enjoy using it.

Reply:
Looks like we can get rid of Tecplot.
What do you think, should we remove it in this PR or should I create an issue and remove it at a later stage?

Reply:
If this can be merged as is that would be good, then I Will make a new PR removing tecplot afterwards.

If you have more on this I Will update tomorrow.

Reply:
Sure!

I'll try to get through this before dinner, but it's probably a good idea to have another pairs of eyes on this.